### PR TITLE
fix(FOROME-1148): fix button in filter refiner

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1148)

### CODE REVIEW

**Comment by AndreyUstyumenko** [comment]
Screen Shot 2022-06-03 at 17 25 14

**Comment by AndreyUstyumenko** [comment]
Screen Shot 2022-06-03 at 17 25 46

File: src/components/undo-redo-buttons/undo-redo-button.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { FC, MouseEvent } from 'react'
```
Comment (AndreyUstyumenko) [inline src/components/undo-redo-buttons/undo-redo-button.tsx]: move to /components

Comment (QSdmitrioul) [reply src/components/undo-redo-buttons/undo-redo-button.tsx]: Fixed

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by AlMaQntr — APPROVED** [review]

**Pull request merged**